tests/src/com/unciv/testing/BasicTests.kt

```diff
@@ -94,7 +94,7 @@ class BasicTests {
             val ruleset = RulesetCache[baseRuleset.fullName]!!
             val modCheck = ruleset.checkModLinks()
             if (modCheck.isNotOK())
-                debug(modCheck.getErrorText(true))
+                debug("%s", modCheck.getErrorText(true))
```

Comment:
With your change `if (params.isEmpty()) msg else msg.format(*params)` this would have been redundant?